0.3.4 #967 cargo install fails

## Minimum reproducible example
cargo install --path mistralrs-server --features cuda

## Error
   Compiling mistralrs-quant v0.3.4 (C:\Users\MATTIA\Desktop\rustsrc\mistral.rs.0.3.4.0967\mistralrs-quant)
   Compiling candle-hf-hub v0.3.3
   Compiling lrtable v0.13.8
   Compiling tokenizers v0.20.4
   Compiling tokio-rayon v2.1.0
   Compiling itertools v0.13.0
   Compiling serde_plain v1.0.2
   Compiling tqdm v0.7.0
   Compiling derive-new v0.7.0
   Compiling derive_more v0.99.18
   Compiling chrono v0.4.38
   Compiling tower-http v0.5.2
error[E0308]: mismatched types
   --> mistralrs-core\src\pipeline\llg.rs:15:67
    |
15  |     let bt = toktrie_hf_tokenizers::ByteTokenizer::from_tokenizer(tokenizer)
    |              ---------------------------------------------------- ^^^^^^^^^ expected `Tokenizer`, found `tokenizers::Tokenizer`
    |              |
    |              arguments to this function are incorrect
    |
    = note: `tokenizers::Tokenizer` and `Tokenizer` have similar names, but are actually distinct types
note: `tokenizers::Tokenizer` is defined in crate `tokenizers`
   --> C:\Users\MATTIA\.cargo\registry\src\index.crates.io-6f17d22bba15001f\tokenizers-0.20.4\src\tokenizer\mod.rs:408:1
    |
408 | pub struct Tokenizer(
    | ^^^^^^^^^^^^^^^^^^^^
note: `Tokenizer` is defined in crate `tokenizers`
   --> C:\Users\MATTIA\.cargo\registry\src\index.crates.io-6f17d22bba15001f\tokenizers-0.21.0\src\tokenizer\mod.rs:408:1
    |
408 | pub struct Tokenizer(
    | ^^^^^^^^^^^^^^^^^^^^
    = note: perhaps two different versions of crate `tokenizers` are being used?
note: associated function defined here
   --> C:\Users\MATTIA\.cargo\git\checkouts\llguidance-fc2ad2787f3486eb\cfef3df\toktrie_hf_tokenizers\src\lib.rs:72:12
    |
72  |     pub fn from_tokenizer(mut hft: Tokenizer) -> Result<ByteTokenizer> {
    |            ^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0308`.
error: could not compile `mistralrs-core` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `mistralrs-server v0.3.4 (C:\Users\MATTIA\Desktop\rustsrc\mistral.rs.0.3.4.0967\mistralrs-server)`, intermediate artifacts can be found at `C:\Users\MATTIA\Desktop\rustsrc\mistral.rs.0.3.4.0967\target`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.

## Other information
Please specify:
Windows 11 Rust 1.83.0
- GPU or accelerator information
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2024 NVIDIA Corporation
Built on Thu_Sep_12_02:55:00_Pacific_Daylight_Time_2024
Cuda compilation tools, release 12.6, V12.6.77
Build cuda_12.6.r12.6/compiler.34841621_0

nvidia-smi
Wed Dec  4 17:01:50 2024
+-----------------------------------------------------------------------------------------+
| NVIDIA-SMI 566.14                 Driver Version: 566.14         CUDA Version: 12.7     |
|-----------------------------------------+------------------------+----------------------+
| GPU  Name                  Driver-Model | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  NVIDIA GeForce RTX 4070 ...  WDDM  |   00000000:01:00.0 Off |                  N/A |
| N/A   48C    P8              2W /   85W |       0MiB /   8188MiB |      0%      Default |
|                                         |                        |                  N/A |
+-----------------------------------------+------------------------+----------------------+

+-----------------------------------------------------------------------------------------+
| Processes:                                                                              |
|  GPU   GI   CI        PID   Type   Process name                              GPU Memory |
|        ID   ID                                                               Usage      |
|=========================================================================================|
|  No running processes found                                                             |
+-----------------------------------------------------------------------------------------+

## Latest commit or version
0.3.4.0967

--- a/mistralrs-core/Cargo.toml
+++ b/mistralrs-core/Cargo.toml
@@ -21,7 +21,7 @@ candle-flash-attn = { git = "https://github.com/EricLBuehler/candle.git", versio
 dirs = "5.0.1"
 hf-hub = { version = "0.3.3", package = "candle-hf-hub" }
 thiserror = "1.0.57"
-tokenizers = "0.20.3"
+tokenizers = "0.21.0"
 tqdm = "0.7.0"
 chrono = "0.4.34"
 minijinja = { version = "2.0.2", features = ["builtins", "json"] }

